Log/v1.5_Changes.md

```diff
@@ -0,0 +1,50 @@
+# Version 1.5 Changes
+
+For many many reasons, we're making some significant changes to the program that will go into effect starting June 19, 2017.
+
+This document outlines the planned changes from a high level. More details to follow.
+
+## What's going away
+
+- Stats (i.e. most of what the "game" formerly was)
+- Levels
+- Wandering in the wilderness (lack of learning tracks)
+
+## New stuff
+
+- Phases (5 of them, designed to be completed in sequence)
+- More support (appropriate to the phase)
+- Phase assessments
+- New membership review process based on assessments
```

Comment:
Suggest adding:

- Tighter scaffolding & structure at earlier phases
- More career prep & support in final phase